Dockerfile

```diff
@@ -26,12 +27,15 @@ RUN apk add --no-cache python3 m4 && \
     sed -i '/.*linux_amd64.zip/!d' terraform_${TERRAFORM_VERSION}_SHA256SUMS && \
     sha256sum -cs terraform_${TERRAFORM_VERSION}_SHA256SUMS && \
     unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin && \
-    curl -L "https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz" | tar -zxf - && \
-    mv linux-amd64/helm /usr/local/bin/helm && \    
+    curl -L "https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz" | tar -zxf - && \
```

Comment:
I would not maintain helmv2 on our ci-aws if we don't pretend to deploy things with Helm2 once we migrate to helm3 on AWS